### PR TITLE
Lecture2

### PR DESCRIPTION
**Pull request by arasevic**
added material for lecture 2

### CODE REVIEW

File: lectures/lecture-02/slides.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Week 1 Slides
```
Comment (colinking) [inline lectures/lecture-02/slides.md]: Week 2

File: lectures/lecture-02/slides.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+{{ 'https://slides.com/arasevic/cmsc389l-week-02' | iframely }}
```
Comment (colinking) [inline lectures/lecture-02/slides.md]: Looks like the first slide still mentions CloudFront!

**Comment by colinking** [comment]
few nits, otherwise lgtm!

File: lectures/lecture-02/slides.md (hunk 1 above)
Comment (arasevic) [reply lectures/lecture-02/slides.md]: fixed in latest commit

File: lectures/lecture-02/slides.md (hunk 2 above)
Comment (arasevic) [reply lectures/lecture-02/slides.md]: fixed

**Comment by colinking** [comment]
lgtm

**Pull request merged**